diff --git a/webview/platform/linux/webview_linux_webkitgtk.cpp b/webview/platform/linux/webview_linux_webkitgtk.cpp
index a90afcc..2700f94 100644
--- a/webview/platform/linux/webview_linux_webkitgtk.cpp
+++ b/webview/platform/linux/webview_linux_webkitgtk.cpp
@@ -1909,7 +1909,7 @@ void Instance::dataRequest(
 		socket->write("HTTP/1.1 ");
 		socket->write(partial ? "206 Partial Content\r\n" : "200 OK\r\n");
 
-		const auto mime = QByteArray(stream->mime());
+		const auto mime = QByteArray::fromStdString(stream->mime());
 		socket->write("Content-Type: " + mime + "\r\n");
 		socket->write("Accept-Ranges: bytes\r\n");
 		socket->write("Cache-Control: no-store\r\n");
